README.md

```diff
@@ -150,32 +150,51 @@ exports.env = {
 };
 ```
 
-### Deploy the webapp
+### Deploy all the apps
+Create a `.env` file in the app directory with the following contents
+```
+COUCH_URL=http://admin:pass@localhost:5984/medic
+COUCH_NODE_NAME=couchdb@localhost
+```
+Then install api and sentinel dependencies
+```
+cd api && npm install && cd ..
+cd sentinel && npm install && cd ..
+```
+Then run
+```
+npm start
+```
+which should start the webapp, api, and sentinel, and watch for changes in each app.
```

Comment:
Replace "which should" with "which will" or "to". "should" implies you have no confidence in it :P

Reply:
Aha yes very good point